hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -2520,6 +2520,16 @@ The switch to turn S3A auditing on or off.
     </description>
 </property>
 
+<property>
+  <name>ipc.server.log.slow.rpc-threshold-ms</name>
+  <value>0</value>
+  <description>The threshold in milliseconds for log slow rpc when ipc.server.log.slow.rpc is enabled.
+    If a request took significant more time than other requests,
+    and its processing time is exceed `logSlowRPCThresholdMs` will consider that as a slow RPC.
+    By default, this parameter is set 0.
+  </description>
```

Comment:
English changes: could we change to the following?

The threshold in milliseconds for **logging** slow rpc when ipc.server.log.slow.rpc is enabled. Besides of being much slower than other RPC requests, an RPC request has to take at least the threshold value defined by this property before it can be considered as slow. By default, this threshold is set to 0 (disabled).